### DISCUSSION TITLE
have the an action option to initiate the actions of a different button

### DISCUSSION THREAD

**Discussion by pchanley** [Budibase/budibase#15257] [Help]
Looking to click the actions of a button after finishing an update via a side panel.

**Comment by ConorWebb96** [comment]
Hey @pchanley,

You won't be able to run these actions automatically. You'll need to click the button after the side panel has closed. However, I believe you can run actions directly on the side panel whenever it is closed, so this may be an option.

**Comment by itribs** [comment]
Hey, I've created a plugin—you can use its binding component for various purposes, such as sharing complex bound values or a group of actions. Depending on your needs, you just need to set up a single binding value and a sequence of actions; whenever you want to trigger those actions, simply update the binding value.

https://github.com/itribs/bb-component-rtools
